io/io/test/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@
 
 ROOT_ADD_GTEST(RRawFile RRawFile.cxx LIBRARIES RIO)
 ROOT_ADD_GTEST(TFile TFileTests.cxx LIBRARIES RIO)
+if (davix)
```

Comment:
This is not strictly necessary: the preprocessor macro `R__HAS_DAVIX` is already available